pkgs/by-name/gc/gcviewer/package.nix

```
@@ -0,0 +1,41 @@
+{ lib, fetchFromGitHub, git, maven, makeWrapper, temurin-jre-bin-17 }:
+let
+  jre = temurin-jre-bin-17;
+in
+maven.buildMavenPackage {
+  pname = "GCViewer";
+  version = "1.37-unreleased";
```

```suggestion
  version = "1.37-unstable-YYYY-MM-DD";
```
with the date of last commit.